cookbooks/oceanic_extension/oceanic_extension.prm

```diff
@@ -0,0 +1,273 @@
+
+#  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 2e6
+set Resume computation                     = auto
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated defect correction Stokes
+set Nonlinear solver tolerance             = 1e-5
+
+# We set the number of nonlinear iterations to 100 as the first 1-2 time steps 
+# often require a large number of iterations to converge.
+# Subsequent time steps should require substantially fewer nonlinear iterations.
+set Max nonlinear iterations               = 100
+
+# Reducing the CFL values and/or maximum time step may improve the convergence behavior 
+# of the Stokes and thermal (advection-diffusion) systems. 
+# However, reducing these values inherently requires more time steps to reach the same end time. 
+# Finding the optimal balance frequently requires testing, although the maximum time step 
+# in lithospheric dynamics simulations with Drucker Prager plasticity is frequently 
+# not set above 20-50 thousand years. 
+set CFL number                             = 0.5
+set Maximum time step                      = 20e3
+set Output directory                       = output_oceanic_extension
+
+# Pressure normalization is turned off because using the free surface implies a surface stress of zero. 
+set Pressure normalization                 = no
+set Adiabatic surface temperature          = 1600
+
+# Governing equations 
+subsection Formulation
+  set Formulation          = Boussinesq approximation
+end
+
+# Model geometry (400x100 km)
+# To extend
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 4
+    set Y repetitions = 1
+    set X extent      = 400e3
+    set Y extent      = 100e3
+  end
+end
+
+# Mesh refinement specification
+# First, globally refine the mesh to 128x32 cells (3.125 km resolution). 
+# Next, adaptively refine the grid to 1.5625 km resolution above 50 km depth 
+# and between horizontal positions of 120 and 280 km. 
+subsection Mesh refinement
+  set Initial global refinement          = 5
+  set Initial adaptive refinement        = 1 
+  set Time steps between mesh refinement = 0
+  set Strategy = minimum refinement function
+  subsection Minimum refinement function
+    set Coordinate system   = cartesian
+    set Variable names      = x,y
+    set Function expression = if (y>50.e3 && x>120.e3 && x<280.e3, 6, 5)
+  end
+end
+
+# Solver parameters
+subsection Solver parameters
+  subsection Stokes solver parameters
+    # Cheap Stokes solver steps are skipped, because the nonlinear rheology 
+    # introduces viscosity contrasts that require the expensive solver.
+    set Number of cheap Stokes solver steps = 0
+    set Linear solver tolerance = 1e-7
+  end
+end
+
+# Free surface advection (vertical or normal)
+subsection Mesh deformation
```

Comment:
This line is hard to see among the comments before and after. Maybe move it to just above the first `set` statement?

Reply:
Moved comments before subsection